nfs-server-and-provisioner/base/class.yaml

```diff
@@ -0,0 +1,7 @@
+kind: StorageClass
```

Comment:
This should go to `cluster-scope` since it's a resource that is cluster scoped and requires cluster admin permissions to apply.